packages/cdk-graph-plugin-diagram/README.md

```diff
@@ -61,9 +61,72 @@ This plugin generates diagrams utilizing the [cdk-graph](https://aws.github.io/a
 
 ### Configuration
 
-See [API Documentation](https://aws.github.io/aws-prototyping-sdk/typescript/cdk-graph-plugin-diagram/index.html) for details, and look in [unit tests](https://github.com/aws/aws-prototyping-sdk/tree/mainline/packages/cdk-graph-plugin-diagram/test/graphviz) for more examples.
+See [API Documentation](https://aws.github.io/aws-prototyping-sdk/typescript/cdk-graph-plugin-diagram/index.html#ipluginconfig) for details, and look in [unit tests](https://github.com/aws/aws-prototyping-sdk/tree/mainline/packages/cdk-graph-plugin-diagram/test/graphviz) for more examples.
```

Comment:
Are we able to also change the example on line 36 to add () at the end of the block as it is currently missing?

Reply:
Fixed